extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -550,28 +550,31 @@ private static function getMediaDetailWithSizeFallback(
 		$fallbackSize = self::MAX_MERCURY_CONTENT_WIDTH
 	) {
 		$mediaDetail = WikiaFileHelper::getMediaDetail( $title, $mediaDetailConfig );
-		if ( empty( $mediaDetail['width'] ) ) {
-			$mediaDetail['width'] = $fallbackSize;
 
-			\Wikia\Logger\WikiaLogger::instance()->error(
-				'ArticleAsJson - Media width was empty - fallback to fallbackSize',
-				[
-					'media_details' => $mediaDetail,
-					'fallback_size' => $fallbackSize
-				]
-			);
-		}
+		if ( !empty( $mediaDetail['mediaType'] ) ) {
+			if ( empty( $mediaDetail['width'] ) ) {
+				$mediaDetail['width'] = $fallbackSize;
+
+				\Wikia\Logger\WikiaLogger::instance()->debug(
+					'ArticleAsJson - Media width was empty - fallback to fallbackSize',
+					[
+						'media_details' => $mediaDetail,
```

Comment:
And when unifying please make it compatible with our API client (Mercury), so use snake_cased in both places.